src/Http/Requests/Log/SearchRequest.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace dnj\ErrorTracker\Laravel\Server\Http\Requests\Log;
+
+use dnj\ErrorTracker\Contracts\LogLevel;
+use Illuminate\Foundation\Http\FormRequest;
+
+class SearchRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            'apps' => ['array', 'nullable'],
+            'apps.*' => ['exists:apps,id'],
```

Comment:
use model name to make it more stable during feature changes.
and when you use model name it's more elegant to use Illuminate\Validation\Rule::exists method.